.github/workflows/avm.res.network.network-watcher.yml

```diff
@@ -0,0 +1,87 @@
+name: "avm.res.network.network-watcher"
```

Comment:
Seams like this snuck in here ;)

Reply:
I don't know where this is coming from 😶‍🌫️